.github/workflows/ci.yml

```diff
@@ -2,27 +2,36 @@
 name: CI
 
 on:
-  - push
```

Comment:
Otherwise, it runs every job twice on every _push_ to a pull request.

What did we need that `push` trigger for in the first place?

Reply:
I can't see the repository setting of `draper`, but if someone allows to commit to the default branch(`master`) directly, we should have this setting to run CI in that case. I think it's better to limit the branch to `master` instead of remove.

```yml
  push:
    branches:
      - master
```

Reply:
Though `master` is protected, I'll include this.